chef/cookbooks/crowbar/templates/default/crowbar-pre-upgrade.sh.erb

```diff
@@ -55,4 +55,48 @@ ovs-vsctl set-fail-mode <%= bridge %> standalone
 echo "Reloading wicked configuration for all interfaces"
 wicked ifreload all
 
+<% if @compute_node %>
+
+echo "Upgrading services needed to act as nova-compute node"
+
+zypper --non-interactive up openstack-nova-compute
+# This is ugly, but fixing neutron spec to upgrade to correct version of pecan is even uglier:
+zypper --non-interactive up python-pecan
+zypper --non-interactive install crudini
+
+echo "Upgrading configuration for Newton based packages"
+
+crudini --set /etc/nova/nova.conf DEFAULT use_neutron true
```

Comment:
So this is apparently already obsolete by latest changes in Newton nova config packages....